### ISSUE TITLE
Stereo brightness issue

### ISSUE THREAD

**Issue by Lextredis** [googlesamples/cardboard-unity#155]
Hello there, I'm struggling with this problem and I can't seem to find a solution:
basically if I play my scene in play mode everything looks fine, but as soon as I build the actual application for android, the brightness of the left-side screen is clearly much higher, and it is pretty noticeable.
I even tried removing every light except for the directional one but the left screen still appears brighter whitout any apparent reason.
The play mode always looks correct, indipendently from any possible configuration.
I tried creating a new project and the problem was still there,
I tried editing an older project with no luck,
I checked every single known issue from the latest version to the very first and found nothing,
Is there any way I can fix this?

Thanks in advance.

**Comment by smdol** [comment]
Can you say which version of Unity, which version of Cardboard SDK, and what your phone model is?  Does this happen with the demo scene too?  Can you capture a screen shot (Volume + Volume Down) and post it?

**Comment by Lextredis** [comment]
Hi, thanks for the answer and sorry for the imprecisions:
The phone is an S4, I am using Unity 5.2.3f1 and the latest version of the cardboard SDK;
I haven't tested the issue on the demo scene, but any new project, even if it's empty, seems to have it.
I have also noticed that the less light there is on a scene the more this issue is visible;
I just tried taking a screenshot but when looking at it from the computer the issue seems gone, while on the actual phone it is still pretty much there, which makes me presum it is an issue on the phone side.
Also, one of the screens is darker than it sould and the other is brighter, none of them are actually rendering correctly.
